dependencies.coffee

```diff
@@ -0,0 +1,37 @@
+child_process = require 'child_process'
+os = require 'os'
+MINIMAL_WIN_CL_VERSION = 15
+spawn = (command) ->
+    [c,a...] = command.split ' '
+    o = child_process.spawnSync c, a
+    o.stdout = o.stdout.toString().trim()
+    o.stderr = o.stderr.toString().trim()
+    o
+
+if os.platform() == 'win32'
+  #http://stackoverflow.com/questions/1233312/finding-version-of-microsoft-c-compiler-from-command-line-for-makefiles#1233332
+  #TODO: test on windows
+  spawn("cl").stdout.replace /Version \d+/, (x) ->
+    if parseInt(x.split(' ')[1]) < MINIMAL_WIN_CL_VERSION
+      throw new Error 'You need Visual Studio 2013 Community Edition to compile ZMQ'
+
+
+# if os.platform() == 'linux'
+
+if os.platform() == 'darwin'
+  unless spawn("brew list | grep 'pkg-config'").stdout.length
+    throw new Error 'You need pkg-config to install zmq: brew install pkg-config'
+  unless spawn("brew list | grep 'zeromq'").stdout.length
```

Comment:
We can rely on the Python `import zmq` test below to check whether zeromq is installed. As above, we don't want to just ask brew whether it has zeromq installed, because it may come from somewhere else too.